### PR TITLE
[OMNIML-2336] make w4a8_nvfp4_fp8's scale factor in range of 448/6

### PR DESCRIPTION
**Pull request by sychen52**
## What does this PR do?
This PR goes together with: https://github.com/NVIDIA/TensorRT-LLM/pull/8180
**Type of change:**  Bug fix
**Overview:** 
For w4a8 nvfp4 fp8, export the scale factor in range of [-448/6, 448/6].

## Usage
export in huggingface_example.sh

## Testing
Test export in model together with import in trtllm

## Before your PR is "*Ready for review*"

- **Make sure you read and follow [Contributor guidelines](https://github.com/NVIDIA/TensorRT-Model-Optimizer/blob/main/CONTRIBUTING.md)** and your commits are signed.
- **Is this change backward compatible?**: Yes/No 
- **Did you write any new necessary tests?**: Yes/No
- **Did you add or update any necessary documentation?**: Yes/No
- **Did you update [Changelog](https://github.com/NVIDIA/TensorRT-Model-Optimizer/blob/main/CHANGELOG.rst)?**: Yes/No 

## Summary by CodeRabbit

* **New Features**
  * Added explicit support for the W4A8_NVFP4_FP8 quantization format so exported models use the correct weight scaling for that format.

* **Bug Fixes**
  * Fixed incorrect fallback behavior for W4A8_NVFP4_FP8 scaling retrieval, improving quantization accuracy and stability during model export.

### CODE REVIEW

File: modelopt/torch/export/quant_utils.py [hunk 1]
```diff
@@ -270,12 +270,16 @@ def get_weight_scaling_factor(module: nn.Module, weight_name: str = "weight") ->
         QUANTIZATION_NVFP4_AWQ,
         QUANTIZATION_W4A8_NVFP4_FP8,
     ]:
+        if quantization_format == QUANTIZATION_W4A8_NVFP4_FP8:
+            # wsf2 for w4a8 needs to be amax/448, so that the wsf is in range 448/6.
+            # This is because the kernel dequantizes weight to fp8, which is in range 448.
+            wsf2 = weight_quantizer._amax.float() / 448.0
```
Comment (cjluo-nv) [inline modelopt/torch/export/quant_utils.py]: let use weight_scaling_factor_2 for readability

**Review by meenchen — APPROVED** [review]
